install.sh

```diff
@@ -489,6 +498,39 @@ StandardError=
 StandardError=inherit
 EOS
     fi
+    for i in daily restart; do
+        install -d -m755 "$rsystemd"/scylla-housekeeping-$i.service.d
```

Comment:
Are these files present in rpm/deb installs? If so they need to be marked so the package manager can clean them up. Or better fold them into the main file.

Reply:
No, it will create these drop-in files only when install.sh manually executed (when --packaging is not specified).